frontstage/controllers/case_controller.py

```diff
@@ -261,3 +210,61 @@ def validate_case_category(category):
         raise InvalidCaseCategory(category)
 
     logger.debug('Successfully validated case category', category=category)
+
+
+def get_cases_for_list_type_by_party_id(party_id, list_type='todo'):
+    logger.debug('Get cases for party for list', party_id=party_id, list_type=list_type)
+
+    business_cases = get_cases_by_party_id(party_id)
+    cases = filter_cases_by_case_group(business_cases)
+    history_statuses = ['COMPLETE', 'COMPLETEDBYPHONE', 'NOLONGERREQUIRED']
+    if list_type == 'history':
+        filtered_cases = [business_case
+                          for business_case in cases
+                          if business_case.get('caseGroup', {}).get('caseGroupStatus') in history_statuses]
```

Comment:
We're doing the safe `get` here. Is there a need though? I'd imagine every case coming through here will have `caseGroup` and `caseGroupStatus`. So we could just have:
`business_case['caseGroup']['caseGroupStatus']`